apps/desktop/src/lib/barmenuActions/ReloadMenuAction.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+	import { listen } from '$lib/backend/ipc';
+	import { createKeybind } from '$lib/utils/hotkeys';
+	import { onMount } from 'svelte';
+
+	onMount(() => {
+		const unsubscribeReload = listen<string>('menu://view/reload/clicked', () => {
+			location.reload();
+		});
```

Comment:
Can we shorten this to just `unsubscribe` since the whole file is about reloading?